python/ml4ir/base/pipeline.py

```diff
@@ -390,6 +390,26 @@ def run(self):
                 experiment_tracking_dict, orient="index", columns=["value"]
             ).to_csv()
 
+            # temperature scaling
+            if self.args.execution_mode in {
+                ExecutionModeKey.TRAIN_INFERENCE_EVALUATE,
+                ExecutionModeKey.TRAIN_EVALUATE,
+                ExecutionModeKey.TRAIN_INFERENCE,
+                ExecutionModeKey.TRAIN_ONLY,
+                ExecutionModeKey.INFERENCE_EVALUATE_RESAVE,
+                ExecutionModeKey.EVALUATE_RESAVE,
+                ExecutionModeKey.INFERENCE_RESAVE,
+            }:
+                if 'calibration' in self.model_config:
```

Comment:
[Minor] @marjanhs You should probably add "calibration" as a string under `ml4ir/base/config/keys` probably ModelConfigKeys? Should be fine for now though.

Reply:
changed accordingly.